tests/utilities_test.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2015 The Switch Authors. All rights reserved.
+# Licensed under the Apache License, Version 2, which is in the LICENSE file.
+
+import unittest
+
+import switch_mod.utilities as utilities
+
+
+class UtilitiesTest(unittest.TestCase):
+
+    def test_approx_equal(self):
+        assert not utilities.approx_equal(1, 2)
+        assert not utilities.approx_equal(1, 1.02)
+        assert utilities.approx_equal(1, 1.01)
+        assert utilities.approx_equal(1, 1)
+
```

Comment:
I tried adding another few tests. The tests pass, but the error message from the exception is printed to the screen. I'd rather hide the expected error messages when testing code that is intended to detect and report errors by raising exceptions. 
```python
    def test_min_data_check(self):
        from pyomo.environ import AbstractModel, Set, Param
        mod = AbstractModel()
        utilities._add_min_data_check(mod)
        mod.set_A = Set(initialize=[1, 2])
        mod.paramA_full = Param(mod.set_A, initialize={1: 'a', 2: 'b'})
        mod.paramA_empty = Param(mod.set_A)
        mod.min_data_check('set_A', 'paramA_full')
        self.assertIsInstance(mod.create(), AbstractModel)
        mod.min_data_check('set_A', 'paramA_empty')
        # This prints the error message to the screen
        self.assertRaises(ValueError, mod.create)
        # So does this
        # with self.assertRaises(ValueError) as err:
        #     mod.create()
        # I don't know how to keep the error message from printing.
```